modules/virt-defining-storageclass.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-preparing-cdi-scratch-space.adoc
+
+[id="virt-defining-storageclass_{context}"]
+= Defining a storage class
+
+To specify the storage class that the Containerized Data Importer (CDI) uses when allocating scratch space, add the `spec.scratchSpaceStorageClass` field to the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+
+* Install the OpenShift client (`oc`).
+
+.Procedure
+
+. Edit the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Add the `spec.scratchSpaceStorageClass` field to the CR, setting the value to the name of a storage class that exists in the cluster:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  scratchSpaceStorageClass: "<storage_class>" <1>
+----
+<1> Replace `<storage_class>` with the storage class that you want to use to provision scratch space for CDI operations in your cluster. If you do not specify a value, scratch space is allocated from the storage class used by the PersistentVolumeClaim (PVC) that is being populated.
```

Comment:
I am not sure you need the first sentence. The introduction and the name of the parameter make it clear that this is for scratch space for CDI. 

Suggestion: <1> If you do not specify a storage class, the CDI uses the storage class of the persistent volume claim that is being populated.